docs/intro/fees.md

```diff
@@ -7,19 +7,19 @@ hide_title: false
 
 Participants who place orders onto the order book, provide liquidity and run the network infrastructure receive the fees that traders pay on filled orders and transfers. The protocol does not charge gas fees for interacting with the network.
 
-You can offset some of those fees, or earn even more, by receiving discounts or getting rewards based on trading activity. Rewards can be funded by anyone, and can be in any asset. See the rewards that are currently available on [Console ↗](https://console.vega.xyz/#/rewards).
+You can offset some of those fees, or earn even more, by receiving discounts or getting rewards based on trading activity. Rewards can be funded by anyone, and can be in any asset. See the rewards that are currently available in mainnet on [Console ↗](https://console.vega.xyz/#/rewards).
 
 For a deeper dive, see the main [discount and rewards](../concepts/trading-on-vega/discounts-rewards.md) page.
 
-As of February 2024, the current fee for taker trades is 0.045% across all markets. This breaks down as:
+As of April 2024, the current fees for taker trades are:
 
-- 0.03% Infrastructure fee
-- 0.01% Liquidity fee
-- 0.005% Maker fee
+![Fees in April 2024](/img/intro/fees/april-fees.png)
```

Comment:
**thought (non-blocking)**: this could be turned in to a component so that it can display "live" or current data like the network param component. Only useful if this ends up being out of date a lot. The link below does fine assuming they don't